invision-app/app/store/actions.js

```diff
@@ -12,16 +12,14 @@ import {
 const moviesAPI = 'http://localhost:4000/movies';
 export function getMovies(order, genre, search) {
```

Comment:
To simplify **serverRenderer.js**, change the parameter sequence:
```suggestion
export function getMovies(search, order, genre) {
```
Do not forget change all invocations of this function